src/Libraries/CoreNodeModels/ColorRange.cs

```diff
@@ -59,7 +59,7 @@ protected virtual void InitializePorts()
 
         void ColorRange_PropertyChanged(object sender, PropertyChangedEventArgs e)
         {
-            if (e.PropertyName != "IsUpdated")
+            if (e.PropertyName != "CachedValue")
                 return;
```

Comment:
This component is also erroneously using `IsUpdated`, which could potentially cause it to initiate multiple rapid and potentially unnecessary updates.